src/RelayClient.ts

```diff
@@ -392,14 +402,17 @@ class RelayClient extends EnvelopingEventEmitter {
   }
 
   public async estimateRelayTransaction(
-    envelopingRequest: UserDefinedEnvelopingRequest
+    envelopingRequest: UserDefinedEnvelopingRequest,
+    options?: RelayTxOptions
   ): Promise<RelayEstimation> {
+    const signerWallet = options?.signerWallet;
+
     const {
       envelopingTx,
       activeRelay: {
         managerData: { url },
       },
-    } = await this._getHubEnvelopingTx(envelopingRequest);
+    } = await this._getHubEnvelopingTx(envelopingRequest, signerWallet);
```

Comment:
NIT: I wouldn't use an additional variable to store it.
```suggestion
    } = await this._getHubEnvelopingTx(envelopingRequest, options?.signerWallet);
```

Reply:
Done.